src/controllers/payment.js

```diff
@@ -1,3 +1,28 @@
-// import { deleteReviewById } from '../queries/deleteReviewById';
-//import Business from '../queries/Business';
+import Payment from '../queries/Payment';
 
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+export async function handleCharge (req, res) {
+  const token = req.body.paymentData.token;
+
+  const charge = await stripe.charges.create({
+    amount: 999,
+    currency: 'usd',
+    description: 'Example charge',
+    source: token
+  });
+
+  if (charge.object === 'charge') {
+    try {
+      const addClicks = await Payment.addClicks(req.body.business, req.body.clicks);
+
+      res.json(addClicks);
+    } catch (error) {
+      // fail!
+      console.log('failed to add to db');
```

Comment:
at least finish the request and send something to indicate to the frontend that something has failed